README.md

```diff
@@ -22,6 +22,7 @@ Meetings are also recorded and posted to the [OpenSSF YouTube channel](https://w
 
 ## Members
 
+<<<<<<< Updated upstream
```

Comment:
merge conflict?

Reply:
oh whoops, thanks, this file isn't supposed to be included :-)